README.md

```diff
@@ -190,6 +190,21 @@ end
 
 Scripttags are created in the same way as the Webhooks, with a background job which will create the required scripttags.
 
+If you want to use the application.js file as generated by the Rails asset pipeline, you can do so by moving the `ShopifyApp.configure`
```

Comment:
`application.js` probably isn't the file you want to send as the script tag. Can you edit this line to just refer to bundles process by the rails asset pipeline